src/components/NavigationBlocker/DefaultFormNavigationBlocker.tsx

```diff
@@ -0,0 +1,50 @@
+import i18n from '@dhis2/d2-i18n'
+import {
+    Modal,
+    ModalActions,
+    ModalContent,
+    ModalTitle,
+    ButtonStrip,
+    Button,
+} from '@dhis2/ui'
+import React from 'react'
+import { useFormState } from 'react-final-form'
+import { useBlocker } from 'react-router-dom'
+
+export const DefaultFormNavigationBlocker = () => {
```

Comment:
Wasn't my intention to add this actually - it's not used in any form yet. So I can remove from this PR and add later if we want.